Makefile

```diff
@@ -17,9 +17,14 @@ endif
 # Targets
 #--------------------------------------------------
 install: 
+	@echo "Installing local runner"
 	@(cd ./local-runner && npm install)
+	@echo "Installing R env packages"
+	@(cd ./pipeline-runner && R -e "renv::restore()")
 build: 
 	@(cd ./local-runner && npm run build)
+test:
+	@(cd ./pipeline-runner && R -e "devtools::test()")
```

Comment:
is this working? we had some problems with @cosa65 running the install script.

Reply:
it works for me but I had most of the deps already installed. We can merge and debug it later

Reply:
yep, I agree. we solved it by manually installing the specific packages that were failing

Reply:
can those missing packages be installed also with a command? like `R -e "install.packages(...)`